docs/source/technical_tutorials/visualization/stac_ipyleaflet.ipynb

```diff
@@ -143,17 +143,17 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Draw an Area of Interest & Get Coordinates\n",
-    "\n",
-    "1. **Activate the Draw Tools** (click on the top Draw button).\n",
-    "2. **Use your mouse** to click, hold and draw a polygon over the map - releasing to finish.\n",
-    "- ![stac_ipyleaflet aoi draw](../../_static/stac_ipyleaflet_draw.png)\n",
-    "\n",
-    "3. **Note** the AOI's **Coordinates & BBox** within the open window.\n",
-    "4. **Print** the AOI's bbox from within a cell:    \n",
-    "    - ![stac_ipyleaflet aoi bbox](../../_static/stac_ipyleaflet_bbox.png)\n",
-    "\n",
-    "5. **Clear** the AOI polygon graphic & coordinates by clicking the **Clear AOI Polygon** Button.\n"
+    "### Interact with the Map\n",
+    "\n",
+    "1. **Activate the Inspect Tools** (click on the top _Inspect_ button)\n",
+    "2. From within the `Point` tab: **Use your mouse** to activate the Point tool; then click on the map at a location of interest\n",
+    "    - **Coordinates** will be printed within the open tab\n",
+    "    - **Raster cell values** will be identified and printed, if raster layers are on\n",
+    "3. From within the `Area` tab: **Use your mouse** to activate the Polygon tool; then click, hold and draw a polygon over the map - releasing to finish\n",
+    "    - The area of interest's **Coordinates & BBox** within the open tab\n",
+    "    - Alternatively **Print** the area of interest's bbox from within a cell:    \n",
+    "    ![stac_ipyleaflet aoi bbox](../../_static/stac_ipyleaflet_bbox.png)\n",
+    "4. **Clear** the point or polygon graphics as needed\n"
```

Comment:
💭 : I can't remember.. did we want to mention how this does not yet work for `/mosaic` endpoints or its safe to leave that out for now since we are working on that this sprint?

Reply:
I think it's ok to leave out since that should be soon resolved.